policy/policy-authority-data.go

```diff
@@ -155,15 +172,10 @@ func (padb *PolicyAuthorityDatabaseImpl) allowedByWhitelist(host string) bool {
 func (padb *PolicyAuthorityDatabaseImpl) CheckHostLists(host string, requireWhitelisted bool) error {
```

Comment:
It seems like you probably want the the same comment here as on allowedByBlacklist above, explaining the different errors that can be returned.